tool_requirements.py

```diff
@@ -6,4 +6,5 @@
 # and inserted into the documentation.
 __TOOL_REQUIREMENTS__ = {
     'verilator': '4.028',
+    'verible': 'v0.0-368-g8177dd8',
```

Comment:
The `v` should probably not be part of the version number.

And there's currently no check added to `check_requirements.py`? Can you file a follow-up for that? (I wouldn't do it as part of this PR, as there are changes to this script in Ibex IIRC. Yes, I know ...)

Reply:
jup, done #2132